localization/english/culture/wc_cultures_l_english.yml

```
@@ -0,0 +1,402 @@
+ mantid_collective_noun:1 "Mantid"
+
+ #Dragon group
+ dragon_group:0 "Dragon"
+ dragon_group_collective_noun:1 "Dragons"
+ green_dragon:0 "Green Dragon"
+ green_dragon_collective_noun:1 "Green Dragons"
+ red_dragon:0 "Red Dragon"
+ red_dragon_collective_noun:1 "Red Dragons"
+ blue_dragon:0 "Blue Dragon"
+ blue_dragon_collective_noun:1 "Blue Dragons"
+ black_dragon:0 "Black Dragon"
+ black_dragon_collective_noun:1 "Black Dragons"
+ bronze_dragon:0 "Bronze Dragon"
+ bronze_dragon_collective_noun:1 "Bronze Dragons"
+
+ #Undead group
+ undead_group:0 "Undead"
+ undead_group_collective_noun:1 "Undead"
+ scourge:0 "Scourge"
+ scourge_collective_noun:1 "Scourged"
+ forsaken:0 "Forsaken"
+ forsaken_collective_noun:1 "Forsaken"
+
+ #Ancient group
+ ancient_group:0 "Ancient"
+ ancient_group_collective_noun:1 "Ancients"
+ protector:0 "Protector"
+ protector_collective_noun:1 "Protectors"
+ ancient_war:0 "War Ancient"
+ ancient_war_collective_noun:1 "War Ancients"
+ ancient_lore:0 "Lore Ancient"
+ ancient_lore_collective_noun:1 "Lore Ancients"
+ ancient_wind:0 "Wind Ancient"
+ ancient_wind_collective_noun:1 "Wind Ancients"
+
+ #Trogg group
+ trogg_group:0 "Trogg"
+ trogg_group_collective_noun:1 "Troggs"
+ trogg:0 "Trogg"
+ trogg_collective_noun:1 "Troggs"
+ grummle:0 "Grummle"
+ grummle_collective_noun:1 "Grummles"
+
+ #Hozen group
+ hozen_group:0 "Hozen"
+ hozen_group_collective_noun:1 "Hozen"
+ forest_hozen:0 "Forest Hozen"
+ forest_hozen_collective_noun:1 "Forest Hozen"
+ jungle_hozen:0 "Jungle Hozen"
+ jungle_hozen_collective_noun:1 "Jungle Hozen"
+ mountain_hozen:0 "Mountain Hozen"
+ mountain_hozen_collective_noun:1 "Mountain Hozen"
+
+ #Goblin group
+ goblin_group:0 "Goblin"
+ goblin_group_collective_noun:1 "Goblins"
+ goblin:0 "Goblin"
+ goblin_collective_noun:1 "Goblins"
+ blackwater:0 "Blackwater"
+ blackwater_collective_noun:1 "Blackwaters"
+ gilblin:0 "Gilblin"
+ gilblin_collective_noun:1 "Gilblins"
+
+ #Kobold group
+ kobold_group:0 "Kobold"
+ kobold_group_collective_noun:1 "Kobolds"
+ kobold:0 "Kobold"
+ kobold_collective_noun:1 "Kobolds"
+ snobold:0 "Snobold"
+ snobold_collective_noun:1 "Snobolds"
+
+ #Draenei group
+ draenei_group:0 "Draenei"
+ draenei_group_collective_noun:1 "Draenei"
+ draenei:0 "Draenei"
+ draenei_collective_noun:1 "Draenei"
+
+ #Gnoll group
+ gnoll_group:0 "Gnoll"
+ gnoll_group_collective_noun:1 "Gnolls"
+ gnoll:0 "Gnoll"
+ gnoll_collective_noun:1 "Gnolls"
+ wolvar:0 "Wolvar"
+ wolvar_collective_noun:1 "Wolvar"
+
+ #Murloc group
+ murloc_group:0 "Murloc"
+ murloc_group_collective_noun:1 "Murlocs"
+ murloc:0 "Murloc"
+ murloc_collective_noun:1 "Murlocs"
+ gorloc:0 "Gorloc"
+ gorloc_collective_noun:1 "Gorlocs"
+
+ #Tuskarr group
+ tuskarr_group:0 "Tuskarr"
+ tuskarr_group_collective_noun:1 "Tuskarr"
+ tuskarr:0 "Tuskarr"
+ tuskarr_collective_noun:1 "Tuskarr"
+
+ #Shath'yar group
+ shathyar_group:0 "Shath'yar"
+ shathyar_group_collective_noun:1 "Shath'yar"
+ nraqi:0 "N'raqi"
+ nraqi_collective_noun:1 "N'raqi"
```

It's their main culture.
```suggestion
 nraqi:0 "Shath'yar"
 nraqi_collective_noun:1 "Old Ones"
```